src/tests/system/tests/test_sss_override.py

```diff
@@ -115,17 +117,20 @@ def test_sss_overrides__overriding_group_name_and_gid(client: Client, provider:
     assert result.gid == 999999, "Local override gid does match original value! "
 
     group.add(name="o-group1", gid=888888)
-
     result = client.tools.getent.group("group1")
     assert result is not None, "Group not found!"
     assert result.gid == 888888, "Group gid does not match override value!"
+    result = client.tools.getent.group("o-group1")
+    assert result is not None, "Group not found!"
+    assert result.gid == 888888, "Group gid does not match override value!"
```

Comment:
This is duplicated below.

Reply:
Did you mean to lookup by original name?

Reply:
I think it's important to assert that the override applies to both IDs in case one ID is used, but yes, it is intentional.

Reply:
I think we can keep old assert messages to make patch smaller.

Reply:
I changed the logic and the docstring and shortened the test.